Source/Core/Core/PatchEngine.cpp

```diff
@@ -156,9 +156,7 @@ void LoadPatches()
   ActionReplay::LoadAndApplyCodes(globalIni, localIni);
 
   // lil silly
```

Comment:
Yeah, I think it was referring to the load and set. Either way it's a pretty pointless comment, so I'll remove it.